### ISSUE TITLE
Routes in Laravel 5.3

### ISSUE THREAD

**Issue by MauroB45** [Haehnchen/idea-php-laravel-plugin#69]
Hi,

Routes autocompletion stop working with Laravel 5.3. Routes file is not longer named 'router.php'.  

Is it any workaround to get this working or should we wait for an update?

Thanks,

**Comment by Haehnchen** [comment]
Looks easy just a split into several subfiles

```
routes/*.php
```

https://mattstauffer.co/blog/routing-changes-in-laravel-5-3
https://laravel.com/docs/5.3/routing#basic-routing

**Comment by MauroB45** [comment]
Hi @Haehnchen,

Is it possible to change the regex as '^.*\/routes\/[\/\\]?(?:.+[\/\\]+?)?(.+?)[\/\\]?$'

That will support inner folder structures inside routes/

**Comment by adreastrian** [comment]
Hello @Haehnchen ,

Is it possible to have what @MauroB45 said?

**Comment by Haehnchen** [comment]
commenting closed issue are hard to follow please open new issue for change / feature requests @adreastrian, @MauroB45. new issue #129
